k8s/utils/health/test_health.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+import cluster_health_check
+from mock import patch, Mock
+
+class Test_Object(object):
+    pass 
+ 
+class TestHealthCheck(unittest.TestCase):
+     
+    def setUp(self):
+        self.node_count = 3
+        pass
+ 
+    def get_node_list_with_conditions(): 
+        test_object = Test_Object()
+        item_array = [Test_Object() for i in range(self.node_count)]
+        conditions = ["Ready", "Not Ready", "Maybe"]
+        for item in item_array:
+            item.status = Test_Object()
```

Comment:
can we break this function into smaller once?

Reply:
Broken the function into two smaller functions.